Documentation/kubernetes-on-aws-destroy.md

```diff
@@ -1,8 +1,8 @@
 ## Destroy the cluster
 
-When you are done with your cluster, run `kube-aws node-pools destroy --node-pool-name <name-of-your-node-pool>` and then `kube-aws destroy` to destroy all the cluster components.
+When you are done with your cluster, run `kube-aws destroy node-pools <name-of-your-node-pool>` and then `kube-aws destroy` to destroy all the cluster components.
```

Comment:
Thanks for the PR but `kube-aws node-pools` sub-command is removed since v0.9.4!
We can't (and won't need to) destroy only node pools. `kube-aws destroy` destroys all the cfn stacks including a control-plane and worker node pool(s).